source/admin_manual/list_of_events.rst

```diff
@@ -14,39 +14,135 @@ See :ref:`event_design` for technical implementation details.
+| maildir            | maildir storage                                         |
++--------------------+---------------------------------------------------------+
+| mbox               | mbox storage                                            |
++--------------------+---------------------------------------------------------+
+| pop3c              | pop3c storage                                           |
++--------------------+---------------------------------------------------------+
+| mailbox            | Mailbox (folder)                                        |
++--------------------+---------------------------------------------------------+
+
+Mailbox Categories
+==================
+
++--------------------+---------------------------------------------------------+
+| Category           | Description                                             |
++--------------------+---------------------------------------------------------+
+| storage            | Mailbox (folder) parent category                        |
++--------------------+---------------------------------------------------------+
+| mail               | Mail                                                    |
++--------------------+---------------------------------------------------------+
+
+Sieve Categories
+================
+
+.. versionadded:: v2.3.11 makes the "sieve" category parent for the other
+                  sieve-* categories.
+
++--------------------+---------------------------------------------------------+
+| Category           | Description                                             |
++--------------------+---------------------------------------------------------+
+| sieve              | Sieve parent category                                   |
++--------------------+---------------------------------------------------------+
+| sieve-action       | Individual Sieve actions executed.                      |
++--------------------+---------------------------------------------------------+
+| sieve-execute      | Sieve script(s) being executed for a particular         |
+|                    | message. This envelops all of Sieve execution; both     |
+|                    | runtime and action execution.                           |
++--------------------+---------------------------------------------------------+
+| sieve-runtime      | Evaluation of individual Sieve scripts.                 |
++--------------------+---------------------------------------------------------+
+| sieve-storage      | Sieve storage                                           |
++--------------------+---------------------------------------------------------+
+
+SQL Categories
+==============
+
++--------------------+---------------------------------------------------------+
+| Category           | Description                                             |
++--------------------+---------------------------------------------------------+
```

Comment:
should use `+===+` for header separator.

Reply:
Fixed